### PR TITLE
Remove Duplicate Include

### PR DESCRIPTION
**Pull request by Shockblast**
Also solves a problem with AnimatedSprites, when you add sprites for the animation the engine crash (Windows 10 x64 I do not know if in the other OS too)

(Edit)
Sorry but it still crashes after this, when I remove the "extra include" and compile, the Load resource-> open on file png for animatedSprite had worked again, but crash again :c.

### CODE REVIEW

File: scene/2d/animated_sprite.cpp [hunk 1]
```diff
@@ -29,8 +29,7 @@
 /*************************************************************************/
 #include "animated_sprite.h"
 #include "os/os.h"
-#include "scene/scene_string_names.h"
-#include "scene/scene_string_names.h"
+#include "scene/scene_string_names.h
```
Comment (Noshyaar) [inline scene/2d/animated_sprite.cpp]: Unmatched double quote here.

Comment (Shockblast) [inline scene/2d/animated_sprite.cpp]: wops

**Review by ghost — APPROVED** [review]

**Comment by akien-mga** [comment]
Could you please rebase to squash the two commits together? http://docs.godotengine.org/en/latest/community/contributing/pr_workflow.html#mastering-the-pr-workflow-the-rebase

**Comment by Shockblast** [comment]
ok, i "rebase" the PR... That was good or I was wrong about something? (sorry my english)

**Comment by Noshyaar** [comment]
Looks good to me. If there's only one commit in the Commits tab and all the changes are listed in the Changes tab then it's okay.

**Pull request merged**